src/models/tiles/text/text-plugin-info.ts

```diff
@@ -7,11 +6,12 @@ export interface ITextPluginInfo {
   iconName: string;
```

Comment:
I didn't have time to fix this API. It seems like it would be better if it matched the other plugins where you provide the toolbar button component (or list of buttons since in this case there are 3 buttons for 1 plugin) and let it handle the icon, tooltip, behavior on click, enabled/disabled state, etc.